sdk/azidentity/azure_cli_credential.go

```diff
@@ -58,8 +63,9 @@ func NewAzureCLICredential(options *AzureCLICredentialOptions) (*AzureCLICredent
 	}
 	cp.init()
 	return &AzureCLICredential{
-		tokenProvider: cp.tokenProvider,
-		tenantID:      cp.TenantID,
+		additionallyAllowedTenants: cp.AdditionallyAllowedTenants,
```

Comment:
How come this one doesn't get the `resolveAdditionallyAllowedTenants()` treatment?

Reply:
I can't think of a reason for it not to, so I believe this is an oversight on the part of past me. Thanks for spotting it.